cfgov/unprocessed/apps/owning-a-home/js/explore-rates/rate-checker.js

```diff
@@ -16,18 +16,22 @@ import median from 'median';
 import unFormatUSD from 'unformat-usd';
 import { applyThemeTo } from './highcharts-theme';
 import { getSelection } from './dom-values';
-import { uniquePrimitives } from '../../../../js/modules/util/array-helpers';
 import {
   calcLoanAmount,
   renderAccessibleData,
   renderDatestamp,
   renderLoanAmount
 } from './util';
+import { uniquePrimitives } from '../../../../js/modules/util/array-helpers';
```

Comment:
This was a consequence of a rebase, but I suppose as a "u" it fits better here. They're all out of alphabetical anyway though.